tgui/packages/tgui/interfaces/BrigTimer.js

```diff
@@ -0,0 +1,121 @@
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { Button, LabeledList, Dropdown, Box, Section } from '../components';
+import { Window } from '../layouts';
+
+export const BrigTimer = (props, context) => {
+  const { act, data } = useBackend(context);
+  data.nameText = data.occupant;
+  if (data.timing) {
+    if (data.prisoner_hasrec) {
+      data.nameText = (<Box color="green">{data.occupant}</Box>);
+    } else {
+      data.nameText = (<Box color="red">{data.occupant}</Box>);
+    }
+  }
+  data.nameIcon = "pencil-alt";
```

Comment:
Define this as its own var here with `let`. Dont modify `data`, its meant to be read only.